eng/pipelines/templates/steps/build.yml

```diff
@@ -48,12 +48,28 @@ steps:
   - template: /eng/pipelines/templates/steps/set-artifact-packages.yml
     parameters:
       Artifacts: ${{ parameters.Artifacts }}
+  - ${{ if and(eq(variables['System.TeamProject'], 'internal'), in(variables['Build.Reason'], 'IndividualCI', 'BatchedCI')) }}:
+    - task: AzureCLI@2
+      inputs:
+        azureSubscription: 'Azure SDK Artifacts'
+        scriptType: 'bash'
+        scriptLocation: 'inlineScript'
+        inlineScript: |
+          az account show
+          expiry=`date -u -d "45 minutes" '+%Y-%m-%dT%H:%MZ'`
+          sasToken=$(az storage container generate-sas --account-name azuresdkartifacts --name azure-sdk-for-js-rush-cache --permissions dlrw --auth-mode login --as-user --expiry $expiry --https-only -o tsv)
```

Comment:
Does the cache require a sastoken? I think we may have disabled access tokens completely from this account.

Reply:
The dynamically generated sas token looks working in the run.

Reply:
OK just learned that this SAS token is based on MSI (https://eng.ms/docs/products/azure-storage/security/standards/identity-based-access#msi-and-delegation-sas) and as such doesn't require an account key. So I think we are safe here.